### PR TITLE
Simsensor docs

### PR DESCRIPTION
**Pull request by PNieck**
Documentation of simsensors

### CODE REVIEW

File: devices/sensors/simsensors.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+|---------------|----|
+| End scenario  |  0 |
+| Accelerometer |  1 |
+| Barometer     |  2 |
+| GPS           |  4 |
+| Gyroscope     |  8 |
+| Magnetometer  | 16 |
+| Thermometer   | 32 |
+
+ - The second column contains a timestamp of a particular measurement. The timestamp should be in **milliseconds**. It is important to note that it does not have a real-time timestamp. The precise meaning of a timestamp is described in the section `Measurement publishing algorithm`.
```
Comment (niewim19) [inline devices/sensors/simsensors.md:29]: Did you mean
```suggestion
 - The second column contains a timestamp of a particular measurement. The timestamp should be in **milliseconds**. It is important to note that it does not have to be a real-time timestamp. The precise meaning of a timestamp is described in the section `Measurement publishing algorithm`.
```
?

**Review by niewim19 — COMMENTED** [review]
I think that the following behavior should be described:
 - ending scenario vs looping the scenario
 - timestamp monotonicity (and what happens if it is not preserved)

File: devices/sensors/simsensors.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+Running selected Simsensor is analogical to start collecting data from a physical device. There is only one required
+argument: the path to a file containing the measurement scenario to be used. Command to start sensor should look file
+this:
+
+```bash
+./sbin/sensors -s <sensor_type>_sim:<scenario_file_path>
+```
+
+From perspective of data consumer, nothing is changing.
```
Comment (niewim19) [inline devices/sensors/simsensors.md:153]:
```suggestion
From perspective of data consumer, nothing changes.
```

File: devices/sensors/simsensors.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+### Starting simsensor
+
+Assume that mocking barometer data is needed and path to file with data to be published is
+`./etc/mock_measurements.csv`. Then the command to start looks like this:
+
+```bash
+./sbin/sensors -s baro_sim:<scenario_file_path>
+```
```
Comment (niewim19) [inline devices/sensors/simsensors.md:162]:
```suggestion
Assume that mocking barometer data is needed and path to file with data to be published is
`./etc/mock_measurements.csv`. Then the command to start looks like this:

```bash
./sbin/sensors -s baro_sim:./etc/mock_measurements.csv
```
```

**Comment by damianloew** [comment]
Could we merge it?

**Review by niewim19 — APPROVED** [review]
Yea, feature is not used, yet present. Documentation should be so.

**Review by damianloew — COMMENTED** [review]
It would be nice to add it also to the table of contents:
Screenshot from 2023-11-24 15-59-20

**Comment by PNieck** [comment]
> It would be nice to add it also to the table of contents: Screenshot from 2023-11-24 15-59-20

OK, I am going to add this

**Review by damianloew — APPROVED** [review]
Thanks, LGTM

**Pull request merged**